monai/transforms/post/array.py

```diff
@@ -858,3 +871,97 @@ def __call__(self, image: NdarrayOrTensor) -> torch.Tensor:
         grad = torch.cat([grad_h, grad_v], dim=1)
         grad, *_ = convert_to_dst_type(grad.squeeze(0), image_tensor)
         return grad
+
+
+class GetInstanceLevelSegMap(Transform):
```

Comment:
Since it is a specific transform for HoVerNet, maybe it's better to move it to `monai/apps/pathology/transforms/post/array.py`
Also for the naming, I would suggest something like `CalcualteInstanceSegmentationMap`.

Reply:
Agreed, have moved it and renamed, thanks.